src/MarioKart8/Cheats/TrophyCheat/rules.txt

```diff
@@ -0,0 +1,171 @@
+[Definition]
+titleIds = 000500001010EC00,000500001010ED00,000500001010EB00
+name = Gold Trophy / 3 stars
+path = "Mario Kart 8/Cheats/Grand Prix Trophies"
+description = Cheat in Grand Prix Trohpies!|Must complete or quit a Grand Prix to save changes.
```

Comment:
If you have a better sugestion for the `Must complete or quit a Grand Prix to save changes.` line
or Better names for `Change EVERYTHING (200/mirror/150/100/50)` `Change Entire Engine Class` `Change Specific Cup (activates by cursor moving onto 50cc)`
please let me know. I think they're fine but feels like they could be worded better